doc/templates.rst

```diff
@@ -76,8 +76,22 @@ Twig templates have access to variables provided by the PHP application and
 variables created in templates via the :doc:`set <tags/set>` tag. These
 variables can be manipulated and displayed in the template.
 
-Use a dot (``.``) to access attributes of a variable (methods, properties
-or constants of a PHP object, or items of a PHP array):
+Twig tries to abstract PHP types as much as possible and works with a few basic
+types, supported by ``filters``, ``functions``, and ``tests`` among others:
+
+=========  ===============================================
+Twig Type  PHP Type
+=========  ===============================================
+string     A sequence of characters or a Stringable object
+number     An integer or a float
+boolean    ``true`` or ``false``
+null       ``null``
+iterable   A sequence, a mapping, or an iterable object
```

Comment:
Doesn't Twig also distinguish sequences and mappings?

Reply:
On the PHP side `sequence` and `mapping` should probably be replaced with `array`?

Reply:
You're right, updated.